package.json

```diff
@@ -2,6 +2,7 @@
   "name": "nva-frontend",
   "version": "0.1.0",
   "private": true,
+  "proxy": "https://bibsys.alma.exlibrisgroup.com",
```

Comment:
Dette løser CORS-problemer når vi skal hente data fra ALMA lokalt. Så lenge vi ikke kan gå via egen backend må ha ha en eller annen workaround på dette, og jeg tror dette er den enkleste måten for nå